sched/scheduler/stateful_scheduler.go

```diff
@@ -406,25 +424,100 @@ func (s *statefulScheduler) step() {
 	s.killJobs()
 	s.scheduleTasks()
 
-	remaining := 0
-	waitingToStart := 0
+	s.updateStats()
+}
+
+/*
+ * update the stats monitoring values:
```

Comment:
Nit that we are mixing use of "//", "/* ... */", and "/* * * * * */" styles

Reply:
fixed